builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -470,10 +470,11 @@ def updateSwaggerConfig() {
  * @param  repoName
  * @return
  */
-def cleanup(repoName) {
+def cleanup(serviceId) {
 
 	try {
-		scmModule.deleteProject(repoName)
```

Comment:
Why are we skipping deleting the project?

Reply:
Including delete project code.